rq_dashboard/cli.py

```diff
@@ -106,6 +109,9 @@ def run(
     RQ_DASHBOARD_SETTINGS environment variable.
 
     """
+    if path:
+        sys.path = path.split(':') + sys.path
```

Comment:
Do we really need to add this to the beginning of sys.path? Maybe `sys.path += path.split(':')` will be more appropriate?
Also I don't really like the concept of providing list of directories as sole string, maybe It will be better to use something like this (not sure about parameter names though):
```
@click.option('--extra-path', multiple=True, help="Append specified directories to sys.path")
```
@voith what do you think?